fragment.go

```diff
@@ -518,13 +518,13 @@ func (f *Fragment) SetFieldValue(columnID uint64, bitDepth uint, value uint64) (
 	for i := uint(0); i < bitDepth; i++ {
 		if value&(1<<i) != 0 {
 			if c, err := f.setBit(uint64(i), columnID); err != nil {
-				return changed, err
+				return changed, errors.Wrap(err, "setting")
```

Comment:
i'm confused about the level of detail necessary.
wouldn't a wrapped error here look like `setting field value: setting`?
in that case, it seems like this should be `setting bit`
perhaps even better: `"setting bit %d", i`

Reply:
I think your first reaction is right, this is overly detailed